x-pack/plugins/security_solution/public/resolver/store/actions.ts

```diff
@@ -24,6 +24,35 @@ interface UserBroughtProcessIntoView {
   };
 }
 
+/**
+ * Dispatched to notify state that a different panel needs to be displayed
+ */
+interface AppDisplayedDifferentPanel {
+  readonly type: 'appDisplayedDifferentPanel';
+  /**
+   * The name of the panel to display
+   */
+  readonly payload: string;
+}
+
+/**
+ * When an examination of query params in the UI indicates that state needs to
+ * be updated to reflect the new selection
+ */
+interface AppDetectedNewIdFromQueryParams {
+  readonly type: 'appDetectedNewIdFromQueryParams';
+  readonly payload: {
+    /**
+     * Used to identify the process the process that should be synced with state.
+     */
+    readonly process: ResolverEvent;
+    /**
+     * The time (since epoch in milliseconds) when the action was dispatched.
+     */
+    readonly time: number;
```

Comment:
Why is `time` included here?

Reply:
take it out and make sure it doesn't break anything